reference/physics.md

```diff
@@ -51,8 +51,15 @@ indicated in the specifications of the robot, then it is more accurate to
 specify the mass rather than the density.
 
 - The `centerOfMass` field defines the position of the center of mass of the
-solid. It is expressed in meters in the relative coordinate system of the
-[Solid](solid.md) node. If `centerOfMass` field is different from [0 0 0], then
+solid.
+If there is no vectors in the `centerOfMass` field, then the center of mass of
+the solid is automatically computed based on the [Solid](solid.md)
+`boundingObject`.
+If there are vectors in the `centerOfMass` field, then the first vector
```

Comment:
Maybe we could also specify that only zero or one vector is supported.
It is not possible to have multiple vecotors in the centerOfMass field.